templates/bloodlineList.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * @var System $system
+ * @var array $ranks
+ * @var array $bloodlines
+ * @var array $RANK_NAMES
+ */
+
+?>
+<script type="text/javascript">
+    function toggleBloodlineDetails(name, byID = false) {
+        if(byID) {
+            name = '#'+name;
+        } else {
+            name = '.'+name;
+        }
+
+        if(name.indexOf("_rank") >= 0) {
+            if($(name+'_box').is(':checked')) {
+                $(name).show();
+            }else {
+                $(name).hide();
+            }
+        } else {
+            $(name).toggle();
+        }
+    }
+</script>
+
+<table class="table">
+    <tr><th>Bloodline List</th></tr>
+    <tr><td>
+        <p>Using the form below, you can search for bloodlines based on rank.</p>
+        <div style="text-align:center;">
+            <?php FOREACH($ranks as $id=>$name): ?>
```

Comment:
```suggestion
            <?php foreach($ranks as $id=>$name): ?>
```
Let's do lowercase on all these